src/hu_stuff.c

```diff
@@ -1534,123 +1347,42 @@ void HU_Drawer(void)
     HU_Erase();
   }
 
+  HUlib_resetAlignOffsets();
+
+  if (message_list)
+    HUlib_drawMText(&w_rtext, align_text);
+  else
+    HUlib_drawSText(&w_message, align_text);
+
+  HUlib_drawSText(&w_secret, align_text);
```

Comment:
Should be `align_direct` for the secret message, right?

Reply:
Pretty sure yes!